AHBC_MIDTERM_2019_JULY_TEAMROCKET/IntegerValidator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AHBC_MIDTERM_2019_JULY_TEAMROCKET
+{
+    class IntegerValidator
+    {
+        private string _inputString { get; set; }
+        private bool _outputInt { get; set; }
+
+        public bool Validate(string intInQuestion)
+
+        {
+            bool temp = int.TryParse(intInQuestion, out int result);
+            if (!temp)
+            {
+                Console.Clear();
+                Console.Write("That is not a valid option. ");
+            }
+
+            return temp;
+
+
```

Comment:
Everything in this looks good, my one question would be.. Should we modify the loop to if they do not input a valid int they will be re prompted to try again? You could do that in this function

Reply:
Oh never mind i see you doing that in the main run app